scripts/constant/cmdAssets.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
@cryptoDev222 why do we have a hard-coded file? Instead we need to fetch dynamically based on folder structure.

Reply:
What we need is a contract name, not folder structure, I think it seems to be hard to get it.
To get it, we should parse every file.